mantidproject/mantid
# waiting mode checks in ISIS direct inelastic reduction

**Description of work.**

Fixes #29765 
Its unclear why the reduction was failing in the first place. 
Here I have added check for archive copying log. 



**To test:**

Probably just check if all new and old tests are passing. Any issues may arise from undefined file system state, and the work should add some clarity.

 


*This does not require release notes* because **it is specfic ISIS reduction workflow issue and the people who are affected will be aware anyway**




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
